src/test/java/com/github/lombrozo/testnames/rules/ml/ModelSourceFileSystemTest.java

```diff
@@ -25,27 +25,41 @@
 
 import java.io.IOException;
 import java.nio.file.Path;
+import java.util.concurrent.atomic.AtomicReference;
+import opennlp.tools.postag.POSModel;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
 
 /**
  * Tests for {@link ModelSourceFileSystem}.
  * @since 0.10
- * @todo #1:90min Enable ML tests when caching will be implemented.
- *  For not tests related to ML take lots of time and it's hard to develop in that mode.
- *  We have to implement a caching or increase the speed of tests in order to use them in
- *  every-day development.
  */
 @Disabled
 final class ModelSourceFileSystemTest {
 
+    /**
+     * Cache.
+     */
+    private static final AtomicReference<POSModel> CACHE = new AtomicReference<>();
+
+    @BeforeAll
+    static void setUp() throws IOException {
+        final ModelSource model = new CachedModelSource(
+            new ModelSourceInternet(), ModelSourceFileSystemTest.CACHE
+        );
+        if (ModelSourceFileSystemTest.CACHE.get() == null) {
+            model.model();
+        }
+    }
+
     @Test
     void loadsFromFileSystem(@TempDir final Path temp) throws IOException {
         final Path path = temp.resolve("model.bin");
-        new ModelSourceInternet().model().serialize(path);
+        ModelSourceFileSystemTest.CACHE.get().serialize(path);
```

Comment:
@h1alexbel The usage of `public static` variables is considered as an anti-pattern. You can read more about it here:

https://stackoverflow.com/questions/51894749/best-practice-to-access-static-object-public-static-or-private-static-with

https://www.yegor256.com/2015/07/06/public-static-literals.html